src/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@300;400;700&display=swap');
+body {
+  font-family: 'Ubuntu', sans-serif;
+  margin: 0 auto;
+}
+$bgc: teal;
+$headerAndFotterColor: rgb(221, 170, 17);
+$searchItemsColor: rgb(23, 203, 216);
```

Comment:
say you want to use teal for a button border. Won't it be confusing for the variable to say bgc? YOu can take a look at this article here https://medium.com/@stefanmorris/proper-naming-of-color-variables-683264fec0f4